src/components/Entities/EntityExecutionsBarChart.tsx

```diff
@@ -88,6 +106,13 @@ export const EntityExecutionsBarChart: React.FC<EntityExecutionsBarChartProps> =
         }
     );
 
+    const handleClickItem = React.useCallback(item => {
+        if (item.metadata) {
+            // const executionId = item.metadata as WorkflowExecutionIdentifier;
```

Comment:
@csirius Maybe, is this commented out by intention?

Reply:
@Pianist038801 
Yes, actually I added routing mechanism for the item click but for next week's demo, we shouldn't include it so just commented it out.